src/components/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,116 @@
+import { useFormik } from "formik";
+import { loginSchema } from "../../../../validationSchemas";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+function Login() {
+  const navigate = useNavigate();
+  const [error, setError] = useState("");
+
+  const formik = useFormik({
+    initialValues: {
+      email: localStorage.getItem("email") || "",
+      password: localStorage.getItem("password") || "",
```

Comment:
this is not ok we cannot store the users password. Imagine someone logs in on the computer then someone else comes and checks the localstorage